tfhe/src/c_api/shortint/parameters.rs

```diff
@@ -47,6 +41,7 @@ pub struct ShortintPBSParameters {
     pub encryption_key_choice: ShortintEncryptionKeyChoice,
 }
 
+#[allow(clippy::fallible_impl_from)]
 impl From<ShortintPBSParameters> for crate::shortint::ClassicPBSParameters {
     fn from(c_params: ShortintPBSParameters) -> Self {
```

Comment:
maybe we should have a try from here then ?

Reply:
Fixed